tests/components/utility_meter/test_sensor.py

```diff
@@ -167,6 +171,7 @@ async def test_device_class(hass):
                 "source": "sensor.energy",
             },
             "gas_meter": {
+                "net_consumption": True,
```

Comment:
Is this just for the test here? Because `gas_meter` should also get a `status_class=STATE_CLASS_TOTAL_INCREASING` shouldn't ?

Reply:
For all devices that can measure both decreasing and increasing values.
Changed it to energy_meter since it is more realistic to have an energy meter that can both increase and decrease.